### PR TITLE
Collection of Logs via Azure Monitor

### PR DESCRIPTION
**Pull request by joinimran**
I have added a section to collect the logs using Azure Monitor Service.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/2376

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/information-protection/windows-information-protection/collect-wip-audit-event-logs.md [hunk 1]
```diff
@@ -163,16 +163,40 @@ Use Windows Event Forwarding to collect and aggregate your WIP audit events. You
 
 2.	In the console tree under **Application and Services Logs\Microsoft\Windows**, click **EDP-Audit-Regular** and **EDP-Audit-TCB**.
 
+## Collect WIP audit logs using Azure Monitor
+You can collect audit logs using Azure Monitor. See [Windows event log data sources in Azure Monitor.](https://docs.microsoft.com/en-us/windows/security/information-protection/windows-information-protection/collect-wip-audit-event-logs)
 
+**To view the WIP events in Azure Monitor**
+1.  Use existing or create new Log Analytics Workspace.
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/windows-information-protection/collect-wip-audit-event-logs.md]:
```suggestion
1.  Use an existing or create a new Log Analytics workspace.
```

File: windows/security/information-protection/windows-information-protection/collect-wip-audit-event-logs.md [hunk 2]
```diff
@@ -163,16 +163,40 @@ Use Windows Event Forwarding to collect and aggregate your WIP audit events. You
...
 
+2.  In Log Analytics->Advanced Settings, go to Data, in Windows Event Logs, add logs to receive:
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/windows-information-protection/collect-wip-audit-event-logs.md]:
```suggestion
2.  In Log Analytics->Advanced Settings, select Data. In Windows Event Logs, add logs to receive:
```

File: windows/security/information-protection/windows-information-protection/collect-wip-audit-event-logs.md [hunk 3]
```diff
@@ -163,16 +163,40 @@ Use Windows Event Forwarding to collect and aggregate your WIP audit events. You
...
+```
+Microsoft-Windows-EDP-Application-Learning/Admin
+Microsoft-Windows-EDP-Audit-TCB/Admin
+```
+>[!NOTE]
+>The Event logs names can be found if using "Windows Events", go to Events folder and go to Properties of the event (Application and Services Logs\Microsoft\Windows, click EDP-Audit-Regular and EDP-Audit-TCB)
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/windows-information-protection/collect-wip-audit-event-logs.md]:
```suggestion
>If using Windows Events Logs, the event logs names can be found under Properties of the event in the Events folder (Application and Services Logs\Microsoft\Windows, click EDP-Audit-Regular and EDP-Audit-TCB)
```

File: windows/security/information-protection/windows-information-protection/collect-wip-audit-event-logs.md [hunk 4]
```diff
@@ -163,16 +163,40 @@ Use Windows Event Forwarding to collect and aggregate your WIP audit events. You
...
 
+3.  Download Microsoft [Monitoring Agent.](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/agent-windows#install-the-agent-using-dsc-in-azure-automation)
 
+4.  To get MSI for Intune installation, as stated in Azure Monitor article, please extract: MMASetup-.exe /c /t:
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/windows-information-protection/collect-wip-audit-event-logs.md]:
```suggestion
4.  To get MSI for Intune installation as stated in the Azure Monitor article, please extract: MMASetup-.exe /c /t:
```

File: windows/security/information-protection/windows-information-protection/collect-wip-audit-event-logs.md [hunk 5]
```diff
@@ -163,16 +163,40 @@ Use Windows Event Forwarding to collect and aggregate your WIP audit events. You
...
+Install Microsoft Monitoring Agent to WIP devices using Workspace ID and Primary key. Workspace ID and Primary key info can be received from "Log Analytics->Advanced Settings"
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/windows-information-protection/collect-wip-audit-event-logs.md]:
```suggestion
Install Microsoft Monitoring Agent to WIP devices using Workspace ID and Primary key. Workspace ID and Primary key info can be found in **Log Analytics** > **Advanced Settings**.
```

File: windows/security/information-protection/windows-information-protection/collect-wip-audit-event-logs.md [hunk 6]
```diff
@@ -163,16 +163,40 @@ Use Windows Event Forwarding to collect and aggregate your WIP audit events. You
...
 
+5.  To deploy MSI via Intune, in installation parameters add: /q /norestart NOAPM=1 ADD_OPINSIGHTS_WORKSPACE=1 OPINSIGHTS_WORKSPACE_AZURE_CLOUD_TYPE=0 OPINSIGHTS_WORKSPACE_ID=<WORKSPACE_ID> OPINSIGHTS_WORKSPACE_KEY=<WORKSPACE_KEY> AcceptEndUserLicenseAgreement=1
 
+>[!NOTE]
+>(Replace <WORKSPACE_ID> & <WORKSPACE_KEY> received from step 5. In installation parameters, don't place <WORKSPACE_ID> & <WORKSPACE_KEY> in quotas "" or '')
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/windows-information-protection/collect-wip-audit-event-logs.md]:
```suggestion
>Replace <WORKSPACE_ID> & <WORKSPACE_KEY> received from step 5. In installation parameters, don't place <WORKSPACE_ID> & <WORKSPACE_KEY> in quotes ("" or '').
```

File: windows/security/information-protection/windows-information-protection/collect-wip-audit-event-logs.md [hunk 7]
```diff
@@ -163,16 +163,40 @@ Use Windows Event Forwarding to collect and aggregate your WIP audit events. You
...
 
+6.  After agent deployed, data will be received within some 10 minutes.
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/windows-information-protection/collect-wip-audit-event-logs.md]:
```suggestion
6.  After the agent is deployed, data will be received within approximately 10 minutes.
```

File: windows/security/information-protection/windows-information-protection/collect-wip-audit-event-logs.md [hunk 8]
```diff
@@ -163,16 +163,40 @@ Use Windows Event Forwarding to collect and aggregate your WIP audit events. You
...
 
+7.  To search for logs, go to Log Analytics Workspace->Logs, in search type: Event
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/windows-information-protection/collect-wip-audit-event-logs.md]:
```suggestion
7.  To search for logs, go to **Log Analytics workspace** >**Logs**, and type **Event** in search.
```

**Comment by joinimran** [comment]
All of the suggestions applied.

**Comment by mypil** [comment]
@Justinha - Can you please assist to check, approve and merge the content of this PR based on issue #2376?

Thank you.

**Pull request merged**